include/Game.hpp

```diff
@@ -46,6 +47,10 @@ class Game : public Ogre::WindowEventListener, public Ogre::FrameListener
     Game &operator=(Game const &) = delete;
     Game &operator=(Game &&) = delete;
     ~Game(void);
+	
+	// Size of window
+	static constexpr int const HEIGHT = 1080;
+	static constexpr int const WIDTH = 1920;
```

Comment:
Yeah, well, for that ... I had to know what were the width and the height of the screen, so the position of the elements in the overlay is relative to screen and not in pixels. And as it's supposed to change, i couldn't put the value directly in my code.